src/sundials/sundials_profiler.c

```diff
@@ -333,22 +339,19 @@ SUNErrCode SUNProfiler_GetElapsedTime(SUNProfiler p, const char* name,
 
 SUNErrCode SUNProfiler_Reset(SUNProfiler p)
 {
-  int i                 = 0;
-  sunTimerStruct* timer = NULL;
-
   if (!p) { return SUN_ERR_ARG_CORRUPT; }
 
   /* Reset the overhead timer */
   sunResetTiming(p->overhead);
   sunStartTiming(p->overhead);
 
-  /* Reset all timers */
-  for (i = 0; i < p->map->max_size; i++)
-  {
-    if (!(p->map->buckets[i])) { continue; }
-    timer = p->map->buckets[i]->value;
-    if (timer) { sunResetTiming(timer); }
-  }
+  // /* Reset all timers */
+  // for (i = 0; i < SUNHashMap_Capacity(p->map); i++)
+  // {
+  //   if (!(p->map->buckets[i])) { continue; }
+  //   timer = p->map->buckets[i]->value;
+  //   if (timer) { sunResetTiming(timer); }
+  // }
```

Comment:
Can this commented code be removed?

Reply:
No, it needed to be revised (and I forgot to come back to it looks like). I have fixed it now.